lib/player/stream_video_source.js

```diff
@@ -244,6 +245,10 @@ shaka.player.StreamVideoSource.prototype.configure = function(config) {
       this.applyRestrictions_();
     }
   }
+
+  this.dispatchEvent(shaka.util.FakeEvent.create({
+    'type': 'configurationchanged'
```

Comment:
Can you please document this event at the top?  See SeekRangeChangedEvent above for an example.